### PR TITLE
Add lowercase token filter docs

### PR DESCRIPTION
**Pull request by AntonEliatra**
### Description
Add lowercase token filter docs

### Issues Resolved
Closes https://github.com/opensearch-project/documentation-website/issues/8154

### Version
all

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by vagimeli** [comment]
@udabhas Will you review this PR for technical accuracy? Thank you.

**Comment by vagimeli** [comment]
> @udabhas Will you review this PR for technical accuracy? Thank you.

@udabhas @varun-lodaya This is over a month old. We need tech review approval to move it forward in the documentation process. Please review this week or provide a peer who can review it. Thank you.

File: _analyzers/token-filters/lowercase.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+The `lowercase` token filter is used to convert all characters in the token stream to lowercase, making searches case insensitive.
+
+## Parameters
+
+The `lowercase` token filter can be configured with the following parameter.
+
+Parameter | Required/Optional | Description
+:--- | :--- | :---
+ `language` | Optional | Specifies a language-specific token filter to use for lowercasing. Valid values are: <br>- [`greek`](https://lucene.apache.org/core/8_7_0/analyzers-common/org/apache/lucene/analysis/el/GreekLowerCaseFilter.html) <br>-  [`irish`](https://lucene.apache.org/core/8_7_0/analyzers-common/org/apache/lucene/analysis/ga/IrishLowerCaseFilter.html) <br>-  [`turkish`](https://lucene.apache.org/core/8_7_0/analyzers-common/org/apache/lucene/analysis/tr/TurkishLowerCaseFilter.html). <br> Default is [Lucene’s LowerCaseFilter](https://lucene.apache.org/core/8_7_0/analyzers-common/org/apache/lucene/analysis/core/LowerCaseFilter.html). 
+
```
Comment (natebower) [inline _analyzers/token-filters/lowercase.md:19]: Line 18: Is `english` also an option, or is that the default used by the Lucene filter?

**Review by natebower — COMMENTED** [review]
@kolchfa-aws A few comments and changes. Thanks!

File: _analyzers/token-filters/index.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Token filter | Underlying Lucene token filter|  Description
...
 `min_hash` | [MinHashFilter](https://lucene.apache.org/core/9_10_0/analysis/common/org/apache/lucene/analysis/minhash/MinHashFilter.html) | Uses the [MinHash technique](https://en.wikipedia.org/wiki/MinHash) to estimate document similarity. Performs the following operations on a token stream sequentially: <br> 1. Hashes each token in the stream. <br> 2. Assigns the hashes to buckets, keeping only the smallest hashes of each bucket. <br> 3. Outputs the smallest hash from each bucket as a token stream.
```
[the remark this answers is not in the document]
Comment (kolchfa-aws) [reply _analyzers/token-filters/index.md:42]: Not quite: the default specifies no language and processes English. There are also a couple of other language options. Reworded.

File: _analyzers/token-filters/lowercase.md (hunk 1 above)
Comment (kolchfa-aws) [reply _analyzers/token-filters/lowercase.md:19]: Default without specifying a language.

**Review by kolchfa-aws — APPROVED** [review]

**Pull request merged**